releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,14 +1,13 @@
 ## Microsoft Extensions Azure App Configuration
 
 ### v1.0.0-preview-008920001-990
-
+#### Released on: June 11<sup>th</sup>, 2019
 * Added `TrimKeyPrefix` method to remove prefixes from the list of returned key-values from the config store.
 * Improved durability on Watch, so that transient network errors do not cause the process to detach.
 * Added additional retries on network error when using `ConnectWithManagedIdentity`.
-* Enabled Correlation-Context for telemetry/request-tracing.
-    * ***Telemetry can be disabled by setting the environment variable `AZURE_APP_CONFIGURATION_TRACING_DISABLED` to `True` or `1`***
-    * Added telemetry for the following:
+* Enabled Correlation-Context for request-tracing.
+    * ***Request-Tracing can be disabled by setting the environment variable `AZURE_APP_CONFIGURATION_TRACING_DISABLED` to `True` or `1`***
+    * Added tracing options for the following:
         * Hosting platform.
         * Differentiating Initialization and Watch/Observe requests.
```

Comment:
"Observe" is not a concept exposed in the config provider. Just "Watch" should be clear enough.

Reply:
Removed it.